functional_tests/scylla_operator/test_functional.py

```diff
@@ -896,6 +891,7 @@ def test_scylla_yaml_override(db_cluster, scylla_yaml):  # pylint: disable=too-m
         else:
             dict(props).pop(hh_throttle_option_name)
 
+    time.sleep(5)
```

Comment:
worth to add a comment line to such sleeps, so it's clear without using git-blame

Reply:
Done, added comment.